lib/assets/connection_cell/main.js

```diff
@@ -58,6 +15,7 @@ export function init(ctx, info) {
   }
 
   ctx.handleEvent("update", ({ fields }) => {
+    ctx.root.innerHTML = renderHTML(fields);
```

Comment:
We only send field changes, if the type doesn't change we don't need to re-render the whole thing :)

```suggestion
    if (fields["type"]) {
      ctx.root.innerHTML = renderHTML(fields);
    }
```